src/node_crypto.cc

```diff
@@ -3135,8 +3144,15 @@ void CipherBase::InitIv(const FunctionCallbackInfo<Value>& args) {
   const node::Utf8Value cipher_type(env->isolate(), args[0]);
   ssize_t key_len = Buffer::Length(args[1]);
   const char* key_buf = Buffer::Data(args[1]);
-  ssize_t iv_len = Buffer::Length(args[2]);
-  const char* iv_buf = Buffer::Data(args[2]);
+  ssize_t iv_len;
+  const char* iv_buf;
+  if (args[2]->IsNullOrUndefined()) {
```

Comment:
This could as well be `->IsNull()` for consistency, no strong opinion here.

Reply:
I'd go with `IsNull()` for consistency with the checks in JS land.